release notes 9.0.1 updated with security advisor (#17643)
elastic/logstash: docs/release-notes/index.md

## 9.0.1 [logstash-9.0.1-release-notes]

::::{important}

> The 9.0.1 release contains fixes for potential security vulnerabilities. See our [security advisory](https://discuss.elastic.co/c/announcements/security-announcements/31) for more details.

Reviewer: ```suggestion
The 9.0.1 release contains fixes for potential security vulnerabilities. 
Check out the [security advisory](https://discuss.elastic.co/c/announcements/security-announcements/31) for details.
```

Reviewer: Minor tweaks to use "one sentence per line" best practice, and more inclusive language.
